lib/v2/chsi/router.js

```diff
@@ -1,3 +1,4 @@
 module.exports = (router) => {
     router.get('/kydt', require('./kydt'));
+    router.get('/hotnews', require('./hotnews'));
```

Comment:
```suggestion
    router.get('/hotnews', require('./hotnews'));
    router.get('/kydt', require('./kydt'));
```
Insert new entries in alphabetic order.